[2023-11-20T23:51:35.041+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_DAG.EXTRACT_API_DATA_TO_S3 scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-20T23:51:35.048+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_DAG.EXTRACT_API_DATA_TO_S3 scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-20T23:51:35.048+0000] {taskinstance.py:1361} INFO - Starting attempt 6 of 7
[2023-11-20T23:51:35.058+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): EXTRACT_API_DATA_TO_S3> on 2023-11-19 00:00:00+00:00
[2023-11-20T23:51:35.064+0000] {standard_task_runner.py:57} INFO - Started process 19349 to run task
[2023-11-20T23:51:35.067+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_DAG', 'EXTRACT_API_DATA_TO_S3', 'scheduled__2023-11-19T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_DAG.py', '--cfg-path', '/tmp/tmpwx1i3r2l']
[2023-11-20T23:51:35.069+0000] {standard_task_runner.py:85} INFO - Job 34: Subtask EXTRACT_API_DATA_TO_S3
[2023-11-20T23:51:35.109+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_DAG.EXTRACT_API_DATA_TO_S3 scheduled__2023-11-19T00:00:00+00:00 [running]> on host dfcabe2f681c
[2023-11-20T23:51:35.167+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='devyani.srivastava' AIRFLOW_CTX_DAG_ID='outcomes_DAG' AIRFLOW_CTX_TASK_ID='EXTRACT_API_DATA_TO_S3' AIRFLOW_CTX_EXECUTION_DATE='2023-11-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-19T00:00:00+00:00'
[2023-11-20T23:52:00.053+0000] {logging_mixin.py:154} INFO - Error uploading DataFrame: module 'datetime' has no attribute 'now'
[2023-11-20T23:52:00.063+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T23:52:00.072+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_DAG, task_id=EXTRACT_API_DATA_TO_S3, execution_date=20231119T000000, start_date=20231120T235135, end_date=20231120T235200
[2023-11-20T23:52:00.132+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T23:52:00.146+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
